traceapp/vis.go

```diff
@@ -92,6 +92,11 @@ func (a *App) d3timelineInner(t *appdash.Trace, depth int) ([]timelineItem, erro
 	}
 	for _, e := range events {
 		if e, ok := e.(appdash.TimespanEvent); ok {
+			// Continue to next iteration
+			// if e.Start() or e.End() are empty time values.
+			if e.Start() == (time.Time{}) || e.End() == (time.Time{}) {
+				continue
+			}
```

Comment:
@slimsag thanks for clarification on why we should not ignore `TimespanEvent` - since it might have valuable information we should still display it on the web ui.

Good call on adding `log.Println` - for sake of consistency I think we should use a `bool` variable named `log` perhaps?; pretty much like [`MemoryStore.log`](https://github.com/chris-ramon/appdash/blob/master/store.go#L53) in order to do logging or not.

Also would like to mention that currently a `TimespanEvent` without invalid/zero times will break the UI, so further investigation & solution should be taken place on the frontend.

***

This commit was vital for #99 so visiting `traces/trace_id` did not break due missing times on `TimespanEvent`, but good news is that we have a workaround introduced with https://github.com/chris-ramon/appdash/commit/6d10ff7337cc52905720d49829408920a21f5831; which essentially won't include those schema related Annotations that were not explicit saved by `InfluxDBStore.Collect(...)` but are present on queries result because `InfluxDB` will return all the fields existing for a measurement.